### PR TITLE
68-more-formatting

### PR DESCRIPTION
**Pull request by bjmckenz**
user edit working completely (select role and major), and major edit working (select required courses). Also some CSS work.

### CODE REVIEW

**Review by liannadevries09 — APPROVED** [review]
looks good to me.

File: student-info-system/schooladmin/templates/major.html [hunk 1]
```diff
@@ -91,6 +91,11 @@ <h3>Required Courses:</h3>
         onclick="window.history.go(-1); return false;"
         value="Back"
     />
+     <input id="editm"
```
Comment (oceancypress) [inline student-info-system/schooladmin/templates/major.html:94]: funky indentation

**Review by oceancypress — APPROVED** [review]

**Comment by oceancypress** [comment]
I noticed we have a bunch of css in a .py file. Is there any way we can decouple our css from our py files? (I havent looked into how this is working but just curious)

**Comment by bjmckenz** [comment]
Not easily.this is the idiom for things like td and th where django does not insert a class. May be possible, so can you convert comment to issue for investigation?

**Pull request merged**